### ISSUE TITLE
Create a documentation page

### ISSUE THREAD

**Issue by juanmc2005** [juanmc2005/diart#75]
## Problem

As the codebase grows, it is difficult for a user to keep track of all the features and mix-and-match possibilities available.

## Idea

Create a documentation page with tutorials and an API reference.

**Comment by amritbhat786** [comment]
Yess please. There are parts in the current documentation page which seem slightly disconnected. For eg, in the [https://github.com/juanmc2005/StreamingSpeakerDiarization#add-your-model](Add your model) section, as a first timer I do not understand how to run the code. Specifically because the code loads a "my_model.ckpt" file but I don't know how to create this file. It would be better if this model file could either be referenced in the repo under an "examples" folder or if its very huge, be downloadable for the demo code to run. 
To summarize, the example code should be runnable off the shelf (preferably without any modifications to the code itself.)

**Comment by juanmc2005** [comment]
Hi @amritbhat786,

The "Add your model" section of the README is for custom models (think EEND or ECAPA-TDNN for example). The default pipeline uses [pyannote/segmentation](https://huggingface.co/pyannote/segmentation) and [pyannote/embedding](https://huggingface.co/pyannote/embedding) automatically and you can run it with `diart.stream` or with `RealTimeInference` like it's shown in the README.

On the other hand I think the "Add your model" title is indeed a bit misleading, do you think it would be clearer if it was something like "Custom models"?

**Comment by amritbhat786** [comment]
Hi, thanks for the clarification. Yes, custom models sounds good to me. Also a comment in the README script would help as well.

**Comment by juanmc2005** [comment]
Thanks @amritbhat786 ! I'll make sure to make those changes for the next release.
If you see any other clarifications that you'd like to make in the readme, you can directly create a pull request. That way it's easier for me to quickly accept and include them in the next release :)

**Comment by juanmc2005** [comment]
A first version of a documentation page is already in `develop` and will appear on readthedocs with the v0.9 release.
I'll create a separate issue for tutorials
